core/src/main/scala/dagr/core/tasksystem/Pipeline.scala

```diff
@@ -84,4 +84,9 @@ abstract class Pipeline(val outputDirectory: Option[Path] = None,
 
   /** True if we this pipeline is tracking this direct ancestor task, false otherwise. */
   def contains(task: Task): Boolean = tasks.contains(task)
-}
+
+  /** Override this to true to print all the command line arguments for the pipeline.  This will only print arguments
+    * if the pipeline is the top-level pipeline.
+    */
+  val printPipelineArgs: Boolean = false
```

Comment:
How about instead of this we have a global parameter in the `application.conf` that controls whether or not usages get printed, and default it to true?